docs/pickaxe_run.rst

```diff
@@ -16,7 +16,8 @@ and explain the components of running pickaxe. Generally, pickaxe_run.py operate
 4. Specification of Filters
 
 This document gives the relevant code snippets from a template and expands on existing comments. Additionally, brief 
-examples of relevant inputs will be created. For more detailed descriptions please see :doc:`inputs` and :doc:`filters`.
+examples of relevant inputs will be created. For more detailed descriptions please see :doc:`inputs` and :doc:`api_ref/filters`.
+To create custom filters, see :doc:`custom_filters`.
```

Comment:
We could make this a tip to really draw attention

```rst
... tip::
    To create custom...
```

Reply:
good idea - I'll add that in